endoreg_db/serializers/meta/__init__.py

```
@@ -0,0 +1,19 @@
+from .sensitive_meta_verification import SensitiveMetaVerificationSerializer
+from .video_meta import (
+    VideoMetaSerializer,
+)
+
+__all__ = [
+    "PDFFileForMetaSerializer",
+    "ReportMetaSerializer",
+    "SensitiveMetaDetailSerializer",
+    "SensitiveMetaUpdateSerializer",
+    "SensitiveMetaVerificationSerializer"
+    "VideoMetaSerializer",
```

Pylint (3.3.7)

[error] Undefined variable name 'SensitiveMetaVerificationSerializerVideoMetaSerializer' in __all__

(E0603)